bitnami/thanos/values.yaml

```diff
@@ -288,6 +288,16 @@ querier:
     ##   key:
     ##   certificate:
 
+    ## Create an ingress object for the GRPC service. The options that are
+    ## accepted are identical to the HTTP one listed above
+    ## to the HTTP one above
+    grpc:
+      enabled: false
+      certManager: false
+      hostname: thanos-grpc.local
+      annotations: {}
+      secrets: []
```

Comment:
The `tls`, `extraHosts` and `extraTls` parameters should be (at least) listed. You can leave the commented, but at least they should be listed so other users can identify them.

Reply:
Yes I will do that. Thank you